AssetInformationApi/V1/Boundary/Request/EditAssetRequest.cs

```diff
@@ -0,0 +1,28 @@
+using Hackney.Shared.Asset.Domain;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace AssetInformationApi.V1.Boundary.Request
+{
+    public class EditAssetRequest
+    {
+        public string AssetId { get; set; }
```

Comment:
Please assess if everything should be editable. For example, an identifier shouldn't be editable.